src/assets/i18n/en.json

```diff
@@ -273,6 +273,12 @@
         "label": "Miles remaining to destination",
         "pattern": "should should only be numbers!"
       },
+      "vesselDraft": {
+        "name": "vesselDraft",
+        "label": "vessel Draft",
+        "pattern": "should should only be numbers!",
```

Comment:
... with at most one digit after the decimal separator.

(Try the error with "123.123" which is "only numbers" but still fails the validation)